### PR TITLE
General24

### PR DESCRIPTION
**Pull request by jiisloth**

### CODE REVIEW

File: vj_cms/cms_plugins.py [hunk 1]
```diff
@@ -9,11 +9,54 @@
...
 
 from vj_cms.client import VJClient
-from vj_cms.models import GameInfo, Timetable, Donatebar
+from vj_cms.models import GameInfo, Timetable, Donatebar, PriorityMessages, AnchorLink, TweakSettings
 from datetime import datetime
 
 client = VJClient(settings.VJ_API_URL)
 
+@plugin_pool.register_plugin
+class MobileSpeechbubble(CMSPluginBase):
```
Comment (Uular) [inline vj_cms/cms_plugins.py]: Plugin names should end with `Plugin`

File: vj_cms/cms_plugins.py [hunk 2]
```diff
@@ -9,11 +9,54 @@
...
+    name = 'Mobile Speechbubble'
+    model = CMSPlugin
+    render_template = "vauhtijuoksu/plugins/mobilespeechbubble.html"
+    cache = False
+    allow_children = True
+
+    def render(self, context, instance, placeholder):
+        context = super().render(context, instance, placeholder)
+        return context
```
Comment (Uular) [inline vj_cms/cms_plugins.py:27]: Space between classes please

File: vj_cms/cms_plugins.py [hunk 3]
```diff
@@ -9,11 +9,54 @@
...
+@plugin_pool.register_plugin
+class TweakSettings(CMSPluginBase):
```
Comment (Uular) [inline vj_cms/cms_plugins.py]: Plugin names should end with `Plugin`

File: vj_cms/cms_plugins.py [hunk 4]
```diff
@@ -9,11 +9,54 @@
...
+    name = 'Tweak Settings'
+    model = TweakSettings
```
Comment (Uular) [inline vj_cms/cms_plugins.py:33]: Model name clashes with the plugin class name – one reason to name the plugin `TweakSettingsPlugin`

File: vj_cms/cms_plugins.py [hunk 5]
```diff
@@ -9,11 +9,54 @@
...
+    render_template = "vauhtijuoksu/plugins/tweak_settings.html"
+
+    def render(self, context, instance, placeholder):
+        context = super().render(context, instance, placeholder)
+        return context
+
+@plugin_pool.register_plugin
+class AnchorLink(CMSPluginBase):
```
Comment (Uular) [inline vj_cms/cms_plugins.py]: Plugin names should end with `Plugin`

File: vj_cms/cms_plugins.py [hunk 6]
```diff
@@ -9,11 +9,54 @@
...
+    name = 'Anchor link'
+    model = AnchorLink
```
Comment (Uular) [inline vj_cms/cms_plugins.py:44]: Model name clashes with the plugin class name – one reason to name the plugin `AnchorLinkPlugin`

File: vj_cms/cms_plugins.py [hunk 7]
```diff
@@ -9,11 +9,54 @@
...
+    render_template = "vauhtijuoksu/plugins/anchor.html"
+
+    def render(self, context, instance, placeholder):
+        context = super().render(context, instance, placeholder)
+        return context
+
+@plugin_pool.register_plugin
+class PriorityMessage(CMSPluginBase):
```
Comment (Uular) [inline vj_cms/cms_plugins.py]: Plugin names should end with `Plugin`

**Review by Uular — CHANGES_REQUESTED** [review]
Styles & templates are probably fine, some suggestions to the plugin module

**Comment by jiisloth** [comment]
New generic theme

**Pull request merged**